sunkit_pyvista/plotter.py

```diff
@@ -22,22 +22,22 @@ class SunpyPlotter(pv.Plotter):
     """
     A plotter for 3D data.
 
-    This class inherits `pyvsita.Plotter` so we can provide coordinate-aware plotting.
+    This class inherits ``pyvsita.Plotter`` so we can provide coordinate-aware plotting.
```

Comment:
Why did all of these stop working or did they never work?

Reply:
I don't think they ever worked, but we added the `default_role` so now they error.

Reply:
Can we not link them at all?!

Reply:
yes, I would think that most of them can be linked.